geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/ListRegionCommand.java

```diff
@@ -110,4 +108,17 @@ public Result listRegion(
     }
     return result;
   }
+
+  public static class ListRegionInterceptor extends AbstractCliAroundInterceptor {
```

Comment:
the call of getMember or findMember inside the command will throw an exception if both are set, so we don't really need to do the check here. I admit that this brings the validation to the client side, but it's a bit of a redundant code. I think we can afford to make that check on the server side if it makes the code a little cleaner.

Reply:
I think cleaner is better, too, but throwing inside the command server-side feels a little too close to the "exception as program flow" antipattern.  Exceptions should be exceptional, right?

There's also the issue that such an exception then gets wrapped a couple of times, to result in the output
```
Could not process command due to error. Error occurred while fetching list of regions. : Please provide either "member" or "group" option.
```
This could be improved by a better message in the root exception, but still would have two confusing layers of output for something that is ultimately a User error, not a server-error.

Reply:
You probably did not rebase to the current develop, I just made a change in CommandExecutor to process UserErrorException differently than general exception. That would probably ease your concern. 

The getMemberXXX call would have to throw an exception anyway to catch the cases where some commands do not do that validation in their interceptors.